numpy_challenge.py

```diff
@@ -0,0 +1,71 @@
+# Создайте 3 ваших любимых эррея разными способами
+# Создайте функцию matrix_multiplication, которая принимает 2 матрицы, перемножает их по соответствующим правилам и выдаёт получившуюся матрицу
+def matrix_multiplication(arr1, arr2):
+	import numpy as np
+	arr3 = np.dot(arr1, arr2)
+	return(arr3)
+
+
+# Создайте функцию multiplication_check, которая принимает список с матрицами, и выдаёт True, если они могут быть перемножены друг на друга в порядке, в котором они находятся в списке, и False, если их нельзя перемножить
+def multiplication_check(arrlist):
+  for i in range(len(arrlist)-1):
+    if arrlist[i].shape[1] == arrlist[i+1].shape[0]:
+      check = "TRUE"
+    else:
+      check = "FALSE"
+      return(check)
+  return(check)
```

Comment:
Лучше не задавать True и False в виде строк, для этого в питоне есть отдельный тип. Можно ещё записать чуть компактнее.
```suggestion
def multiplication_check(arrlist):
  for i in range(len(arrlist)-1):
    if arrlist[i].shape[1] != arrlist[i+1].shape[0]:
      return False
  return True
```